### PR TITLE
Explorer new design- transaction

### PR DESCRIPTION
**Pull request by Jibz1**
- Transaction Details page
- ! Not done, thumbnail and Modules pagination 

Screen Shot 2022-07-13 at 12 52 30 PM

https://user-images.githubusercontent.com/8676844/178798036-7875468f-f088-41e0-90e5-cd44fda35492.mov

### CODE REVIEW

**Comment by Jibz1** [comment]
- base on latest design changes
https://user-images.githubusercontent.com/8676844/178816796-8ba21e56-c21d-4c4c-b589-409b29b338cf.mov

**Comment by 666lcz** [comment]
Looks great! A few small nit:

1. it should be "Sender" instead of "sender & recipient) for the `Call` and `Publish` transaction page

CleanShot 2022-07-13 at 15 39 37


2. We are missing the package id, module, function for the move call. Please reference the new design below

CleanShot 2022-07-13 at 15 38 52

3. It looks like the height of the modules should be higher?
CleanShot 2022-07-13 at 15 45 49
CleanShot 2022-07-13 at 15 45 45

**Comment by Jibz1** [comment]
-This is just a temp fix @Andrew47 is working on this page 
Screen Shot 2022-07-13 at 6 59 51 PM

**Comment by Jibz1** [comment]
@666lcz package details 
Screen Shot 2022-07-13 at 7 57 55 PM

**Comment by Jibz1** [comment]
Screen Shot 2022-07-14 at 9 39 26 AM

File: explorer/client/src/pages/transaction-result/TransactionResult.tsx [hunk 1]
```diff
@@ -30,7 +29,7 @@ import type {
     SuiObjectRef,
 } from '@mysten/sui.js';
 
-import styles from './TransactionResult.module.css';
+// import styles from './TransactionResult.module.css';
```
Comment (666lcz) [inline explorer/client/src/pages/transaction-result/TransactionResult.tsx:32]: are we still using this file? If not let's remove it

**Comment by Jibz1** [comment]
- Update code module to display syntax highlight 
- View more added 
https://user-images.githubusercontent.com/8676844/179034103-832f8f53-a5cc-4ce2-b271-735f67e6a00e.mov

**Review by 666lcz — APPROVED** [review]

**Pull request merged**